app/serializers/event_serializer.rb

```diff
@@ -16,4 +16,10 @@ def address
 
     "#{object.address1} #{object.address2}"
   end
+
+  # serialize instances of distance through model
+  attribute :estimated_distance do
```

Comment:
Should this be in the agency serializer as well?

Reply:
I'm planning on it.  It will be in the next PR.

Reply:
Addressed in latest commit